src/tasks/initiate_new_auction.ts

```diff
@@ -84,6 +90,30 @@ const initiateAuction: () => void = () => {
       );
 
       console.log("Using EasyAuction deployed to:", easyAuction.address);
+      if (
+        taskArgs.allowListManager !=
+        "0x0000000000000000000000000000000000000000"
+      ) {
+        const allowListManager = await hardhatRuntime.ethers.getContractAt(
+          "AllowListVerifier",
+          taskArgs.allowListManager,
+        );
+        const interfaceSignatureOfIsAllowed = allowListManager.interface
+          .getSighash("isAllowed(address,uint256,bytes)")
```

Comment:
You could also just use the function name, according to the [docs](https://docs.ethers.io/v5/api/utils/abi/interface/#Interface--selectors). 
```suggestion
          .getSighash("isAllowed")
```